test/fake-timers-test.js

```diff
@@ -3283,7 +3283,7 @@ describe("FakeTimers", function () {
         });
 
         it("mirrors toString", function () {
-            assert.same(this.clock.Date.toString(), Date.toString());
```

Comment:
I am not sure why this was needed? Do they no longer return the same thing? If so, it's a bit strange that they are the same.

Reply:
This is actually now a stronger comparison. So the test is better. We assert that it is the same function